Meditation-App/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Meditation-App</title>
+</head>
+<body>
+    <div class="wrapper">
+        <video class="video" loop src=""></video>
+                <audio class="audio" loop  src="""></audio>
+        
+        <div class="buttons-minutes">
```

Comment:
uninformative classes.
For example, it can be ._setting-time_ and _.setting-time__button_

Reply:
Thank you, fix